### PR TITLE
Display grouped user attribute by scopes in OIDC apps

### PR DESCRIPTION
**Pull request by mpmadhavig**
### Purpose
$subject

### Related Issues
- https://github.com/wso2-enterprise/iam-engineering/issues/229

### Related PRs
- https://github.com/wso2/identity-apps/pull/3497
- https://github.com/wso2/identity-apps/pull/3521
- https://github.com/wso2/identity-apps/pull/3522

### Checklist
- [x] e2e cypress tests locally verified.
- [x] Manual test round performed and verified.
- [x] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### Screenshots

add-user-attributes-list-grouped-user-attributes
_Fig 1: User attribute section_

add-user-attribute-select-user-attribute
_Fig 2: User attribute section scope expanded view_

### CODE REVIEW

File: apps/console/src/features/applications/components/settings/attribute-management/attribute-list-item.tsx [hunk 1]
```diff
@@ -156,7 +190,10 @@ export const AttributeListItem: FunctionComponent<AttributeListItemPropInterface
     }, [ claimMappingOn ]);
 
     return (
-        <Table.Row data-testid={ testId }>
+        <Table.Row data-testid={ testId + "" }>
```
Comment (dasuni-30) [inline apps/console/src/features/applications/components/settings/attribute-management/attribute-list-item.tsx:193]: Any reason to add "" to data-testid?

File: apps/console/src/features/applications/components/settings/attribute-management/attribute-selection-oidc.tsx [hunk 2]
```diff
@@ -0,0 +1,823 @@
+/**
```
Comment (dasuni-30) [inline apps/console/src/features/applications/components/settings/attribute-management/attribute-selection-oidc.tsx:1]: Can we extend the existing attribute-selection component? 
https://github.com/wso2/identity-apps/blob/master/apps/console/src/features/applications/components/settings/attribute-management/attribute-selection.tsx

File: apps/console/src/features/applications/components/settings/attribute-management/attribute-selection-oidc.tsx [hunk 3]
```diff
@@ -0,0 +1,823 @@
...
+            )
+            : null
+    );
+};
+
+/**
+ * Default props for the application attribute selection component.
+ */
+AttributeSelectionOIDC.defaultProps = {
+    "data-testid": "application-attribute-selection"
```
Comment (dasuni-30) [inline apps/console/src/features/applications/components/settings/attribute-management/attribute-selection-oidc.tsx:822]: Can we give a different data-testid?
https://github.com/wso2/identity-apps/blob/master/apps/console/src/features/applications/components/settings/attribute-management/attribute-selection.tsx#L1160

File: modules/react-components/src/components/accordion/segmented-accordion/segmented-accordion-title.tsx [hunk 4]
```diff
@@ -119,7 +119,7 @@ export interface StrictSegmentedAccordionTitleActionInterface {
  *
  * @param props - Props injected to the component.
  *
- * @returns Segmented Accordion Title React Component
+ * @returns
```
Comment (dasuni-30) [inline modules/react-components/src/components/accordion/segmented-accordion/segmented-accordion-title.tsx:122]: Can we add what we are returning?

File: modules/react-components/src/components/accordion/segmented-accordion/segmented-accordion.tsx [hunk 5]
```diff
@@ -35,14 +35,19 @@ export interface SegmentedAccordionSubComponentsInterface {
...
+    viewType?: "list-view" | "table-view";
+}
 
 /**
  * Segmented accordion component.
  *
  * @param props - Props injected to the component.
  *
- * @returns Segmented Accordion React Component
+ * @returns
```
Comment (dasuni-30) [inline modules/react-components/src/components/accordion/segmented-accordion/segmented-accordion.tsx:50]: Can we add what we are returning?

File: apps/console/src/features/applications/components/settings/attribute-management/attribute-list-item.tsx (hunk 1 above)
Comment (mpmadhavig) [reply apps/console/src/features/applications/components/settings/attribute-management/attribute-list-item.tsx:193]: No I guess its a mistake. Will resolve it.

Comment (mpmadhavig) [reply apps/console/src/features/applications/components/settings/attribute-management/attribute-list-item.tsx]: resolved with https://github.com/wso2/identity-apps/pull/3528/commits/acf93bbed6a21c13a9d22d84cc8294ea664ad8c4

File: apps/console/src/features/applications/components/settings/attribute-management/attribute-selection-oidc.tsx (hunk 3 above)
Comment (mpmadhavig) [reply apps/console/src/features/applications/components/settings/attribute-management/attribute-selection-oidc.tsx]: resolved with https://github.com/wso2/identity-apps/pull/3528/commits/acf93bbed6a21c13a9d22d84cc8294ea664ad8c4

File: modules/react-components/src/components/accordion/segmented-accordion/segmented-accordion-title.tsx (hunk 4 above)
Comment (mpmadhavig) [reply modules/react-components/src/components/accordion/segmented-accordion/segmented-accordion-title.tsx]: resolved with https://github.com/wso2/identity-apps/pull/3528/commits/acf93bbed6a21c13a9d22d84cc8294ea664ad8c4

File: modules/react-components/src/components/accordion/segmented-accordion/segmented-accordion.tsx (hunk 5 above)
Comment (mpmadhavig) [reply modules/react-components/src/components/accordion/segmented-accordion/segmented-accordion.tsx]: resolved with https://github.com/wso2/identity-apps/pull/3528/commits/acf93bbed6a21c13a9d22d84cc8294ea664ad8c4

**Review by dasuni-30 — APPROVED** [review]

File: apps/console/src/features/applications/components/settings/attribute-management/attribute-selection-oidc.tsx (hunk 2 above)
Comment (mpmadhavig) [reply apps/console/src/features/applications/components/settings/attribute-management/attribute-selection-oidc.tsx:1]: Will resolve this in a future PR.

**Pull request merged**